### ISSUE TITLE
Search support on webjars.com (patch supplied)

### ISSUE THREAD

**Issue by robshep** [webjars/webjars#765]
Hi,

I find the list of webjars too long for quick navigation.

I made a quick and dirty search form, which bolts into the index page.  Please consider adding it, or something like it.

Thanks Rob

Below this line:

```
<h2 style="float: left">List of WebJars (<a href="/documentation">Docs</a>)</h2>
```    

..... Add the following lines

```
    <div class="pull-left" style="margin: 20px 40px 0px 40px">
        <div class="form-group has-feedback">
          <input type="text" class="form-control" id="search" placeholder="Search">
          <span class="glyphicon glyphicon-remove form-control-feedback" id="clearSearch"  aria-hidden="true" style="top:0;color:#999999;display:none" title="Clear Search"></span>
        </div>
        <script type="text/javascript">
            $(function(){
                $("#search").keyup(function(a,b,c){
                    var searchText = $(this).val();
                    if(searchText == ""){
                        $("#clearSearch").hide();
                    }
                    else {
                        $("#clearSearch").show();
                    }
                    if(searchText.length == 1){
                        return;
                    }
                    filter(searchText)

                })

                $("#clearSearch").click(function(){
                    $("#search").val("")
                    $(this).hide()
                    filter();
                })

                var filter = function(search){
                    console.log()
                    if(!search){
                        $("tr[data-artifact]").show()
                        return;
                    }
                    $("tr[data-artifact]").each(function(){
                        if( $(this).data("artifact").indexOf(search) == -1 ) {
                            $(this).hide();
                        }
                        else {
                            $(this).show()
                            
                        }
                    });
                }
            })
        </script>
    </div>
```

**Comment by jamesward** [comment]
Yes, it is getting long.  But how is this different from Ctrl-F?

**Comment by robshep** [comment]
CTRL-F is the goto page search tool agreed. But I always find it a bit clunky leaping and wrapping around the page on CTRL-G. 
This just consolidates the viewable list to only items of interest. Take it or leave it.

**Comment by jamesward** [comment]
Cool.  Want to send a PR so that you can be credited with the work?

**Comment by robshep** [comment]
No problem.  Is it this view template?
 https://github.com/webjars/webjars/blob/master/app/views/index.scala.html

**Comment by jamesward** [comment]
Fixed by #766.
